clevercloud/siaes_sync_c1_c4.sh

```
@@ -0,0 +1,21 @@
+#!/bin/bash -l
+
+# Fetch new siaes from C1 + update existing
+
+# Do not run if this env var is not set:
+if [[ -z "$CRON_SYNC_C1_C4_ENABLED" ]]; then
+    exit 0
```

Ici ça peut _fail silently_ si cette variable est pas _set_. Du coup n'y a-t-il pas un risque que les tâches tournent sans erreur alors qu'il ne se passe rien ?

---

je me suis inspiré du C1. Mais ca me va que ca fail silently, tant que ca tourne en prod. On va documenter ca

---

j'ai rajouté un echo si la variable d'env n'est pas set